bears/xml/XMLBear.py

```diff
@@ -24,10 +26,11 @@ class XMLBear(LocalBear, Lint):
     gives_corrected = True
 
     def process_output(self, output, filename, file):
-        if self.stdout_output:  # only yield Result if stdout is not empty
-            return self._process_corrected(self.stdout_output, filename, file)
-        if self.stderr_output:  # pragma: no cover
-            return self._process_issues(self.stderr_output, filename)
+        if self.stderr_output:
+            self.use_stderr = True
```

Comment:
@SanketDG if i do `use_stderr = True` without checking whether the stderr_output is there or not it will set ``results_output `` to stderr_output even when there are no any errors in stderr. 
https://github.com/coala-analyzer/coala/blob/master/coalib/bearlib/abstractions/Lint.py#L98